src/service/world-api-adapter.js

```diff
@@ -1,11 +1,22 @@
 /**
  * ##World API Adapter
  *
- * The World API Adapter allows you to create, access, and manipulate multiplayer worlds its users and runs
+ * Epicenter includes support for multiplayer games. These are projects where multiple end users share a run and work together in the same "world." Only [team projects](../../../glossary/#team) can be multiplayer.
  *
- * All API calls take in an "options" object as the last parameter. The options can be used to extend/override the World API Service defaults.
+ * The World API Adapter allows you to create, access, and manipulate multiplayer worlds, including end users and runs playing within the world.
```

Comment:
'runs playing within the world' sounds slightly confusing. 

The World API Adapter allows you to create, access, and manipulate multiplayer worlds within your Epicenter project. You can use this to manage end users playing within the world, and access their runs. <NRN: I think 'manage' users and 'access' runs is the right terminology, given than can you can't really manipulate the run from this adapter>